app/src/main/java/org/simple/clinic/util/moshi/LocalDateMoshiAdapter.kt

```diff
@@ -4,22 +4,34 @@ import com.squareup.moshi.FromJson
 import com.squareup.moshi.ToJson
 import java.time.LocalDate
 import java.time.format.DateTimeFormatter
+import java.time.format.DateTimeParseException
+import java.util.Date
 
 class LocalDateMoshiAdapter {
 
   companion object {
     private val formatter = DateTimeFormatter.ISO_DATE!!
+    private val indiaNHIDFormatter = DateTimeFormatter.ofPattern("d/M/yyyy")
```

Comment:
When defining this date formatter, we also should be explicitly setting the locale to `Locale.ENGLISH` here. Otherwise, we might run into issues if users have the device locale set to something other than non-english.